src/main/java/step2/domain/SplitNumber.java

```diff
@@ -0,0 +1,18 @@
+package step2.domain;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class SplitNumber {
+    private SplitNumber() {
+    }
+
+    public static Set<LottoNumber> getLottoNumbers(String winnersNumbers) {
+        return Arrays.stream(winnersNumbers.split(","))
+                .map(Integer::parseInt)
+                .map(LottoNumber::valueOf)
```

Comment:
```suggestion
                .map(LottoNumber::valueOf)
```
integer로 변경하는 부분도 LottoNumber에게 위임하면 어떨까요?